### PR TITLE
Merge selenium branch

### PR DESCRIPTION
**Pull request by mpsonntag**
This PR adds front end testing via selenium to the `banners` branch. The PR
- adds all files required for basic testing with selenium
- example selenium tests
- a description how to run selenium tests locally
- updates the docker file to also include banner and banner_mobile directories

### CODE REVIEW

**Comment by achilleas-k** [comment]
Is there any reason for `Cookies.py` having an uppercase first letter?  Not a problem, just unconventional.

File: test/frontend/Readme.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+Before the tests can run, several steps are needed in order for them to work.
+
+First of course, you need to install python-selenium, if you haven't done so already, 
+after setting up a virtual environment (not necessarily in this order):
+
+`pip install selenium pytest`
+
+Next, you need to get the drivers for the browsers your interested in, at the moment, 
+Google Chrome and Firefox are used. The drivers can eb downloaded from this page:
```
Comment (achilleas-k) [inline test/frontend/Readme.md]:
```suggestion
Google Chrome and Firefox are used. The drivers can be downloaded from this page:
```

File: test/frontend/Readme.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+session cookies:
+
+```
+python load_cookies.py
+```
+
+Now we can run the tests via the `pytest` or alternatively the `python -m pytest` command. Without any arguments,
+all tests in the directory will be executed.
+Specific tests can always be run by adding the file name to this command.
+Using the `pytest-xdist` module, parallel testing is possible, decreasing the overall run time of the tests.
```
Comment (achilleas-k) [inline test/frontend/Readme.md]: Maybe mention that pytest-xdist should be installed separately (it's not included in pytest).

Comment (mpsonntag) [reply test/frontend/Readme.md]: Good point, will do.

**Pull request merged**